src/Templator.ts

```
@@ -53,7 +53,7 @@ export class Templator {
                 const referencedIds = new Set<string>();
                 this.notes?.forEach(note => {
                     const refs = note.matchData["Links"] || [];
-                    refs.forEach(refId => referencedIds.add(refId));
+                    refs.forEach(refId => referencedIds.add(refId.replace(/[\[\]]/g, '')));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)